services/director-v2/src/simcore_service_director_v2/modules/osparc_variables/substitutions.py

```diff
@@ -248,6 +253,7 @@ async def resolve_and_substitute_session_variables_in_specs(
                     project_id=project_id,
                     node_id=node_id,
                     run_id=service_run_id,
+                    wallet_id="" if wallet_id is None else wallet_id,
```

Comment:
We need to define this somewhere. There is a CLI that displays all osparc variables registered. Perhaps we should add some doc there that claims that if `wallet_id` is `None` , then the variable is still defined but without a value. We need to be very precise in the definitions on the interfaces. 


Check `list_osparc_session_variables` and perhaps we need to extend the registration step such that we can cadd descriptions that can be displayed using the CLI